react/week1/todolist/src/Todo.js

```diff
@@ -0,0 +1,20 @@
+
+
+import React  from 'react'
+
+export default function todo({todo}){
+    return(
+        <div>
+         
+         <ul>
+            
+       <li> {todo.description} </li>
+        
+      <li>  {todo.deadline}</li>
+      </ul>
```

Comment:
The Todo item here should only be wrapped in 1 <li>{todo.description} {todo.deadline}</li>

You could use the <ul></ul> in the parent to wrap all the items